src/store/modules/ui.store.js

```diff
@@ -246,11 +253,15 @@ export default {
         setEmbed({ commit }, { embed, dispatcher }) {
             commit('setEmbed', { embed: !!embed, dispatcher })
         },
-        setShowLoadingBar({ commit }, { loading, dispatcher }) {
-            commit('setShowLoadingBar', { loading, dispatcher })
+        setShowLoadingBar({ commit }, { requester, loading, dispatcher }) {
```

Comment:
I don't know if I like this name for what it does.
Maybe something like `addLoadingBarSource` or something in this veins would help us remember that we need to pass more than the `loading` flag (`setShow` translate as, "give me a flag" to me, no necessarily give me a requester or a source that is loading)

Or maybe even split this functionality into two actions :
  - `addLoadingRequester`
  - `addLoadedRequester`

Reply:
@pakb Good point the naming is not 100% good, what do you think of two actions:
- `dispatch('setLoadingBarRequester', {requester: 'printing-map', dispatcher})`
- `dispatch('clearLoadingBarRequester', {requester: 'printing-map', dispatcher})`
?